app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -49,6 +53,11 @@ class OnboardingFragmentPresenter @Inject constructor(
     onboardingPagerAdapter =
       OnboardingPagerAdapter(fragment.requireContext(), getOnboardingSlideFinalViewModel())
     binding.onboardingSlideViewPager.adapter = onboardingPagerAdapter
+    val isRTL = TextUtilsCompat
```

Comment:
Instead of this implementation we should ideally use `ViewPager2` to fix this issue.
I have filed an issue for the same, please mention that issue number in `TODO` comment so that we can track this.

Something like

`// TODO (#2194 ): Remove this once the implementation is updated to ViewPager2`

Reply:
Done